adeolaidowu/MovieApi
# implementation of cloudinary for update/add movie route

What does this PR do?
This PR sends images to add/updated to cloudinary

 List of what get done in this PR
[x] Implementation of the register and login endpoint.
How can this be tested?
- clone this repository
- Run from visual studio

Expected result
[x] Returns an Ok request if upload to cloudinary is successful
[x] Returns a Bad request if it is unsuccessful
